pkg/policy/execute.go

```diff
@@ -381,6 +385,22 @@ func GenerateExecutionResultFile(result *ExecutionResult, outputDir string) erro
 	return nil
 }
 
+// extractFirstPathComponent extracts the first path component form a given string.
+// e.g: "a/b/c" -> "a"
+//      "a" -> "a"
+//      "a//b" -> "a"
+func extractFirstPathComponent(str string) (string, error) {
```

Comment:
Filepath base or dir?

Reply:
filepath base and dir don’t fit here:
base: get last component
dir: get all except last component